CHANGES.md

```diff
@@ -42,6 +42,7 @@ Features
 * [#612](https://github.com/java-native-access/jna/pull/612): Kernel32Util#freeLocal/GlobalMemory always throws Win32Exception if failed [@lgoldstein](https://github.com/lgoldstein).
 * [#608](https://github.com/java-native-access/jna/pull/608): Mavenize the build process - change parent and native pom artifactId/name to differentiate in IDE and build tools. [@bhamail](https://github.com/bhamail)
 * [#613](https://github.com/java-native-access/jna/pull/613): Make Win32Exception extend LastErrorException [@lgoldstein](https://github.com/lgoldstein).
+* [#613](https://github.com/java-native-access/jna/pull/614): Added standard Kernel32Util#closeHandle method that throws a eption if failed to close the handle [@lgoldstein](https://github.com/lgoldstein).
```

Comment:
throws a .... exception :) Also quote these function names in backticks probably (and above).